xarray/coding/cftime_offsets.py

```diff
@@ -949,7 +953,9 @@ def cftime_range(
     inclusive: InclusiveOptions = "both",
     calendar="standard",
 ) -> CFTimeIndex:
-    """Return a fixed frequency CFTimeIndex.
+    """DEPRECATED: use date_range(use_cftime=True) instead.
+
+    Return a fixed frequency CFTimeIndex.
```

Comment:
This should render a little more nicely and include a link to `xarray.date_range`:
```suggestion
    """Return a fixed frequency CFTimeIndex.

    .. deprecated:: 2025.02.0
        Use :py:func:`~xarray.date_range` with ``use_cftime=True`` instead.
```